libcst/codemod/_cli.py

```diff
@@ -277,91 +277,76 @@ def _parallel_exec_process_stub(  # noqa: C901
         transformer.context = replace(
             transformer.context,
             filename=filename,
-            full_module_name=_calculate_module(repo_root, filename),
+            full_module_name=_calculate_module(config.repo_root, filename),
         )
 
         # Run the transform, bail if we failed or if we aren't formatting code
         try:
             input_tree = parse_module(
                 oldcode,
                 config=(
-                    PartialParserConfig(python_version=python_version)
-                    if python_version is not None
+                    PartialParserConfig(python_version=str(config.python_version))
```

Comment:
Could the `str()` be dropped here?

Reply:
Unfortunately not – Pyre would complain:

```
libcst/codemod/_cli.py:289:40 Incompatible parameter type [6]: Expected `Union[libcst._parser.types.config.AutoConfig, str]` for 1st parameter `python_version` to call `libcst._parser.types.config.PartialParserConfig.__init__` but got `Optional[str]`.
```